plugins/polio/tests/test_lqas_map.py

```diff
@@ -666,7 +666,9 @@ def test_lqas_zoomedin_end_date_filter(self):
         content = json.loads(response.content)
         results = content["results"]
         self.assertEquals(len(results), 3)
-        self.assertEquals(results[0]["data"]["campaign"], self.campaign_1.obr_name)
+        # There's no guarantee on the order.
+        obr_names = [item["data"]["campaign"] for item in results]
+        self.assertIn(self.campaign_1.obr_name, obr_names)
```

Comment:
The test checks that e get the correct round so, although the  order doesn't matter much, but we `obr_name` and `round_number` should be checked together, i.e assertion on next line (672) should be 2 if `obr_name == self.campaign_1.obr_name`